### DISCUSSION TITLE
见怪不怪 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#471] [General]
# 见怪不怪 - Miao Yu | 于淼

https://yufree.cn/cn/2024/07/15/wired/

**Comment by earfanfan** [comment]
>现代社会本质就是一个见怪不怪空前繁荣的时期。其中有三怪，第一为等级怪，凡事皆要有三六九等，分个高下是很多人生存的意义；第二为解释怪，凡事求个逻辑自洽自圆其说，价值观优先，不允许存在解释不了的现象；第三为术语怪，凡事喜欢找专家小圈子，以黑话区别你我抱团取暖，维护自己圈子的利益。这三怪在现代人脑子里根深蒂固，不论一个人嘴上如何说众生平等思想自由求同存异，但落到具体场景里，大都是一副见怪不怪的样子。

啊，三水你这篇文章写得好好玩呀，虽然你提到的好些词语我以前没见过，不过并不妨碍阅读。话又说回来，等级怪不就是大家喜欢通过攀比来得到个体的优越感么，解释怪就是追求某种确切可知、可以控制的安全感，哈哈我在解读也算是解释怪行为，至于术语怪嘛像是前两者的混合体，同时制造标准和用标准去衡量比较。如果是脑中只存在单一价值观的人，用单一的意识形态标准去看待世界就会这样啊。

悄摸地说，我觉得我自己曾经就是等级怪、解释怪、术语怪，真的。我在追求独立思考的路上大概经历了这样的三个阶段。

1. 单一价值观之下，被灌输的意识形态观念洗脑。
2. 质疑评价标准，质疑标准的合理性，却走向不去比较。
3. 不比较会忽视差异的存在，但也要跳出一元化价值观或者二元化的比较，这是现阶段的状态，不知道以后还会不会改变。

话说三水你好久没耕博客了啊，我又想拍你的马屁了，哈，我来拍。如果说这个世界上存在平行宇宙，在无数个平行世界里面，一定是现在的三水你是最搞笑的。那么问题来了，如果是未来的三水穿越到现在这个时刻，哪个三水才是最搞笑的呢？答案还是现在的三水，因为未来的三水会随着年龄增长被世界改变，变得不搞笑了，哈哈哈。那么问题又来了，如果是未来的三水先去到过去把过去的三水也带到了现在，那么到底哪个三水是最搞笑的呢？答案还是现在的三水，因为“现在的三水是最搞笑的”是现在的我拍的马屁，这个马屁在过去三水所在的世界不存在。
